src/components/bulk-email-tool/bulk-email-form/BulkEmailRecepient.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+const DEFAULT_GROUPS = {
+  SELF: 'myself',
+  STAFF: 'staff',
+  ALL_LEARNERS: 'learners',
+  VERIFIED: 'track:verified',
+  AUDIT: 'track:audit',
+};
+
+export default function BulkEmailRecepient(props) {
+  const { handleCheckboxes, selectedGroups } = props;
+  return (
+    <Form.Group>
+      <Form.Label>Send To:</Form.Label>
```

Comment:
Is `Send To:` hardcoded english here?

Reply:
fixed but github doesn't mark as "outdated" cause I renamed this file.